src/Components/Home/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import { Typography } from '@mui/material';
+
+const HomeScreen = () => {
+  return (
+    <Box
+      sx={{
+        width: '99.9%',
+        textAlign: 'center',
+        minHeight: '100vh',
+        overflow: 'hidden',
+      }}>
+      {/* Slider component */}
```

Comment:
please delete this comment. The code should be enougth documentation to itself. Add a div with a name="sliders"

Reply:
Also the slider should be after the welcome title